### PR TITLE
migration for removing password_digest, change validation to reflect

### PR DESCRIPTION
**Pull request by twarbelow**
## Pull request type
Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: 
closes #

## What is the new behavior?

- migrating will no longer add password_digest
- 
- 

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: db/migrate/20210803003329_remove_password_digest_from_users.rb
Comment (mkrumholz) [inline db/migrate/20210803003329_remove_password_digest_from_users.rb:5]: Looks good to me here.

File: db/schema.rb
Comment (mkrumholz) [inline db/schema.rb:38]: I think we want this :uid right? When I ran migrations it was deleted. Wondering if we maybe need to add that in as a migration too?

Comment (mkrumholz) [reply db/schema.rb:38]: @twarbelow thoughts?

**Review by markcyen — APPROVED** [review]
Looks good to me on removing password and leaving in the token. Will defer to others on the uid.

**Review by mkrumholz — APPROVED** [review]
Going to approve and see if I can merge here, just to keep things moving along. We can discuss uid later, if this works.

**Pull request merged**
